ebi_eva_common_pyutils/assembly/assembly.py

```diff
@@ -29,3 +30,17 @@ def get_supported_asm_from_ensembl(tax_id: int) -> str:
     if assembly_accession_attribute in response:
         return str(response.get(assembly_accession_attribute))
     return None
+
+
+@lru_cache
+def get_ensembl_rapid_release_data():
```

Comment:
Considering we are constructing a taxonomy->assembly map, I would suggest changing the name of the function to something more specific or building the map in a different function.